alvr/client_openxr/src/extra_extensions/passthrough_fb.rs

```diff
@@ -49,11 +51,16 @@ impl PassthroughFB {
         let layer = sys::CompositionLayerPassthroughFB {
             ty: sys::CompositionLayerPassthroughFB::TYPE,
             next: ptr::null(),
-            flags: xr::CompositionLayerFlags::EMPTY,
+            flags: xr::CompositionLayerFlags::BLEND_TEXTURE_SOURCE_ALPHA,
             space: sys::Space::NULL,
             layer_handle,
         };
 
+        // HACK: YVR runtime seems to ignore IS_RUNNING_AT_CREATION
+        if platform == Platform::Yvr {
```

Comment:
I already mentioned in another comment, the platform for PFD MR should be split from YVR. You could add a `is_play_for_dream()` utility method which check for either MR or YVR

Reply:
Just to clarify, I ended up doing `is_yvr` because internally the MR is "YVR_D3" (the prior headsets are D2 and D1) and all the OpenXR exts are sticking with YVR as well, Play For Dream seems to only be a US marketing name.